src/axom/core/docs/sphinx/core_acceleration.rst

```diff
@@ -0,0 +1,51 @@
+.. ## Copyright (c) 2017-2021, Lawrence Livermore National Security, LLC and
+.. ## other Axom Project Developers. See the top-level LICENSE file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+******************************************************
+Core acceleration
+******************************************************
+

```

Comment:
We should probably define "accelerators" here to provide the reader with some context.
I.e. this page is about support for different programming model backends, like OpenMP and CUDA.

This page might be helpful: https://raja.readthedocs.io/en/main/feature/policies.html

Reply:
I think the link is now https://raja.readthedocs.io/en/main/sphinx/user_guide/feature/policies.html.

I added some high-level discussion.